README.md

```diff
@@ -20,6 +20,7 @@ Using fluent-plugin-logdna, you can send the logs you collect with Fluentd to Lo
   #mac C0:FF:EE:C0:FF:EE                     # optional mac address
   #ip 127.0.0.1                              # optional ip address
   #tags web,dev                              # optional tags
+  request_timeout_seconds 30                 # optional timeout for upload request, default 30 seconds
```

Comment:
It seems cumbersome to include `seconds` in the config variable itself.

Also looking at the config variable `buffer_chunk_limit`, it supports an arbitrary amount of bytes with a cool suffix system. Do we want to have the same thing for `request_timeout_seconds` so it's instead `request_timeout 30s`. Just a thought

Reply:
My preference is generally for clarity over brevity, particularly since we're not consistent in which time units to use between libraries. But having said that the `buffer_chunk_limit` system in the Fluentd plugin is cool. Let me see about doing that and supporting seconds (s) and milliseconds (ms), defaulting to seconds.

Reply:
Haven't yet seen where Fluentd does this unit parsing for `buffer_chunk_limit`.